src/analysis/normed_space/riesz_lemma.lean

```diff
@@ -0,0 +1,30 @@
+import analysis.normed_space.basic
+import topology.metric_space.hausdorff_distance
+
+variables {k : Type*} [normed_field k]
+variables {α : Type*} [normed_group α] [normed_space k α]
+
+/-- Riesz's Lemma. Stated in terms of multiples of norms since in
```

Comment:
Could you explain the statement in the docstring? I guess Riesz has written several lemmas, and I didn't know what Riesz' lemma referred to before I read the statement below.